tests/fuzzer.c

```diff
@@ -489,6 +489,53 @@ static int basicUnitTests(U32 const seed, double compressibility)
     }
     DISPLAYLEVEL(3, "OK \n");
 
+    DISPLAYLEVEL(3, "test%3d: superblock uncompressable data, too many nocompress superblocks : ", testNb++)
+    {
+        ZSTD_CCtx* cctx = ZSTD_createCCtx();
```

Comment:
nit : `const`ify : `ZSTD_CCtx* const cctx`
Also : check that `cctx` allocation is successful (`!= NULL`).